plugins/powerd/fu-plugin-powerd.c

```diff
@@ -133,5 +152,38 @@ fu_plugin_update_prepare (FuPlugin *plugin,
 			     MINIMUM_BATTERY_PERCENTAGE_FALLBACK);
 		return FALSE;
 	}
-	return TRUE;
+	return fu_plugin_inhibit_suspend (error);
+}
+
+gboolean
+fu_plugin_update_cleanup (GError **error)
```

Comment:
I think you need to rename this function.
Given the context, I think you meant to call this `fu_plugin_delete_suspend_file`.

Although you might want to call it `fu_plugin_powerd_delete_suspend_file` to make it more specific.

Reply:
I'll get on that
I also forgot to do some checks and this doesn't build; I probably won't get changes to fix that in today.